tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/ObjcInteropPrinter.swift

```diff
@@ -174,6 +176,52 @@ internal class ObjcInteropPrinter: BasePrinter {
         writeLine("}")
     }
 
+    private func print(objcInteropNestedAssociatedTypeEnum: ObjcInteropAssociatedTypeEnum) throws {
+        let propertyWrapper = objcInteropNestedAssociatedTypeEnum.parentProperty
+        let swiftProperty = propertyWrapper.bridgedSwiftProperty
+        let className = objcTypeNamesPrefix + objcInteropNestedAssociatedTypeEnum.objcTypeName
+        let rootClassName = objcTypeNamesPrefix + objcInteropNestedAssociatedTypeEnum.objcRootClass.objcTypeName
+
+        if swiftProperty.mutability == .mutable {
+            throw Exception.unimplemented("Generating mutable `ObjcInteropAssociatedTypeEnum` is not supported: \(swiftProperty.type).")
+        }
```

Comment:
This limits the work to only current version of RUM schema, where `action.id` is immutable. Supporting mutability adds some extra work, which we can IMO postpone for now.

Reply:
Agreed 👍